app/components/CompanyValues.tsx

```diff
@@ -46,22 +48,22 @@ export default function CompanyValues() {
             {/* QUALITY VALUE CONTAINER */}
             <HStack spacing={12} align="center">
               <VStack align="start" spacing={4}>
-                <Heading size="lg">Quality</Heading>
+                <Heading size="lg">{props.quality1?.title}</Heading>
                 <Box
                   bgGradient="linear(90deg, #DB0A40 0%, rgba(219, 10, 64, 0.04) 96%, rgba(219, 10, 64, 0) 100%)"
                   w="full"
                   h={1}
                 ></Box>
-                <Text fontWeight="bold" fontSize="lg">
-                  {
-                    "At Kenworthy Machine, producing quality parts to our customer's specifications is our most important job."
-                  }
-                </Text>
-                <Text fontSize="lg">
-                  {
-                    "Our team is dedicated to making sure your parts are correct an in tolerance every time. They use their expertise, as well as modern inspection equipment to achieve this goal. Upon request we can provide inspection reports for your parts with detailed measurements compared to drawing specifications. We also offer laser engraving to give your parts a professional appearance."
-                  }
-                </Text>
+                <TinaMarkdown
+                  content={props.quality1?.description}
+                  components={{
+                    p: ({ children }) => (
+                      <Text as="span" fontSize="xl">
```

Comment:
The text is showing up as white against a white background in my browser.

Reply:
I added a color prop!

Reply:
That fixed it. It looks great now!